pkg/kubelet/status/status_manager_test.go

```diff
@@ -1406,6 +1406,7 @@ func TestMergePodStatus(t *testing.T) {
 		desc                          string
 		enablePodDisruptionConditions bool
 		hasRunningContainers          bool
+		HasUnpreparedResources        bool
```

Comment:
lower case

Reply:
Hopefully this is not needed as we don't need to change tests anymore.